### PR TITLE
Mlflow_tfm_ner_testcase

### PR DESCRIPTION
**Pull request by SatyamRajawat**

### CODE REVIEW

**Comment by SatyamRajawat** [comment]
Hi @AshishMahendra , Please review it.

**Review by AshishMahendra — COMMENTED** [review]
Let's Merge the changes, I'll have to update the test file to use the old testing set. I can create a PR once this is merged

File: jaseci_kit/jaseci_kit/modules/tfm_ner/tfm_ner.py [hunk 1]
```diff
@@ -6,15 +6,16 @@
 import pandas as pd
 import json
 import warnings
-from .train import (
+from train import (
```
Comment (ypkang) [inline jaseci_kit/jaseci_kit/modules/tfm_ner/tfm_ner.py:9]: @AshishMahendra Does this actually work? I remember we had to set the importing to be relative with the `.` to get the deployment options working. Did you test this through all the deployment options, including `kubectl apply -f tfm_ner.yaml` and actions load local/module/remote?

File: jaseci_kit/jaseci_kit/modules/tfm_ner/tfm_ner.py [hunk 2]
```diff
@@ -6,15 +6,16 @@
...
-    train_model,
+    training,
     load_custom_model,
     save_custom_model,
     data_set,
     check_labels_ok,
+    model_versions,
 )
-from .entity_utils import create_data, create_data_new
+from entity_utils import create_data, create_data_new
```
Comment (ypkang) [inline jaseci_kit/jaseci_kit/modules/tfm_ner/tfm_ner.py:18]: same here.

File: jaseci_kit/jaseci_kit/modules/tfm_ner/tfm_ner.py [hunk 3]
```diff
@@ -36,11 +37,17 @@ def config_setup():
         model_config = json.load(jsonfile)
 
     curr_model_path = model_config["model_name"]
+    # staging_model_path = model_config["staging_model_path"]
```
Comment (ypkang) [inline jaseci_kit/jaseci_kit/modules/tfm_ner/tfm_ner.py:40]: Let's not have any commented out code block in the code base. If they are not needed, delete the code. If they should be kept around for some reason, leave a comment describing why.

File: jaseci_kit/jaseci_kit/modules/tfm_ner/train.py [hunk 4]
```diff
@@ -348,45 +398,110 @@ def test_score(test_loader, model):
     return "testing done!"
 
 
-def train(epoch, optimizer, training_loader, max_grad_norm):
-    model.train()
-    train_score(optimizer, training_loader, max_grad_norm)
-    if val_loader is not None:
-        val_score(val_loader, model)
+# function for start model training sepratly
+def start_model_training(
```
Comment (ypkang) [inline jaseci_kit/jaseci_kit/modules/tfm_ner/train.py:402]: I am still curious why these functions are renamed. Do we know why?

File: jaseci_kit/jaseci_kit/modules/tfm_ner/train.py [hunk 5]
```diff
@@ -348,45 +398,110 @@ def test_score(test_loader, model):
...
+        "Total time taken to completed training : ",
+        str(total_time),
+        logs_file_name,
+    )
+    logs(str(datetime.now()) + "    ", "--" * 30, logs_file_name)
 
-# DEFINING MODEL training
-def train_model(
-    model_name, epochs, mode, lab_check, learning_rate, max_grad_norm, model_save_path
+    # ###################### testing started ##########################
```
Comment (ypkang) [inline jaseci_kit/jaseci_kit/modules/tfm_ner/train.py:446]: I am not a fun of these type of comment in the code. It should just be a normal comment `# Testing started.`

**Review by ypkang — CHANGES_REQUESTED** [review]
@AshishMahendra I just realized that I never actually submitted my review comments on this one...here they are. 

Can you do the same thing as you did with the biencoder one -- copy the code to your fork so we can make changes if we need?

**Comment by ypkang** [comment]
Replaced by #359

**Pull request closed without merging**